lib/membership/update.js

```diff
@@ -24,15 +24,29 @@ var Member = require('./member.js');
 var uuid = require('node-uuid');
 var util = require('util');
 
-function Update(address, incarnationNumber, status, localMember) {
-    this.address = address;
-    this.incarnationNumber = incarnationNumber;
-    this.status = status;
+/**
+ * Create a new Update
+ * @param {IMember} subject the subject of the update.
+ * @param {IMember} [source] the source of the update.
+ * @constructor
+ */
+function Update(subject, source) {
+    if (!(this instanceof Update)) {
```

Comment:
It's not used but most other objects in the repo are using a `instanceof`-check to make sure it's invoked using `new`. When cleaning up this class I thought it made sense to add it here as well but if you prefer I can remove it.